kibana/lib/openshift_logging_plugin/index.js

```diff
@@ -0,0 +1,28 @@
+import { resolve } from 'path';
+
+export default function (kibana) {
+  return new kibana.Plugin({
+    name: 'openshift_logging',
+    id: 'openshift_logging',
+    require: ['elasticsearch'],
+
+    uiExports: {
+      
+      hacks: [
+        //the path here is misleading as it matches the 'name' not actual path
+        'plugins/openshift_logging/hack'
```

Comment:
can we call it something other than `hack` ?

Reply:
interceptors is the name I chose